admin/listPurchases/listView.html

```diff
@@ -116,7 +123,7 @@
                 (entry.status == "pending" ?
                     `<div class="mark-as-sent">
                         <div>Når du har afsendt ordren til kunden, kan du markere det her, så sendes kvitteringen.</div>
-                        <a class="btn btn-big mark-as-sent-btn" href="javascript:markOrderDispatched('${entry.id}');">
+                        <a class="btn btn-big mark-as-sent-btn" data-order-id="${entry.id}" href="javascript:markOrderDispatched('${entry.id}');">
```

Comment:
Now you have two different ways to assign the same action. Instead, we should roll back the `href` to its previous value: `href="/admin/orders/${entry.id}/mark-dispatched"` - that way it works without the Javascript listeners attaching.

If you just register two different javascript listeners it isn't really close to progressive enhancement. Consider reading the article I linked you again.